src/components/global/ContactFormModal.jsx

```diff
@@ -0,0 +1,138 @@
+// React Imports
+import { useRef } from 'react';
+// emailjs Imports
+import emailjs from '@emailjs/browser';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+// Other Library Imports
+import { PropTypes } from 'prop-types';
+
+// TODO: Change title from "Contact Us" to "Volunteer" if using that button?
+
+const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) => {
+  const form = useRef();
+
+  const handleClose = () => {
+    setShowContactFormModal(false);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    emailjs.sendForm('service_co2agxf', 'template_20pnwni', form.current, 'AkBl59Ya3226OfPyQ').then(
+      (result) => {
+        console.log(result.text);
+      },
+      (error) => {
+        console.log(error.text);
+      }
+    );
+    e.target.reset();
+  };
+
+  return (
+    <Dialog open={showContactFormModal} onClose={handleClose}>
+      <form ref={form} onSubmit={handleSubmit} autoComplete="off">
+        <Box
+          sx={{
+            p: { xs: 0, md: 4 }
+          }}
+        >
+          <DialogTitle sx={{ display: 'flex', justifyContent: 'center' }}>Contact Us</DialogTitle>
+          <DialogContent>
+            <DialogContentText sx={{ pb: '1rem' }}>
+              For any questions or to just reach out, contact us today!
```

Comment:
maybe just "For any questions or comments, contact us today!"   ?

Just a thought, not an issue the way it is.

Reply:
Or maybe this isn't even needed at all, just have the title?

Reply:
Yeah I'm not sure. This part seems very arbitrary. I'm not sure there are really any "wrong" answers.

I'm fine with pretty much anything, if there's any sort of guide or anyone has a strongly held belief haha